src/accounts/accounts.controller.ts

```diff
@@ -0,0 +1,145 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+  HttpException,
+  ParseUUIDPipe,
+} from '@nestjs/common';
+import { DepositTransactionDto } from 'src/transactions/dto/deposit-transaction.dto';
+import { WithdrawTransactionDto } from 'src/transactions/dto/withdraw-transaction.dto';
+import { SwyftTablesInfo } from 'src/typings/types';
+import { CreateTransactionDto } from '../transactions/dto/create-transaction.dto';
+import { Transaction } from '../transactions/entities/transaction.entity';
+import { AccountsService } from './accounts.service';
+import { CreateAccountDto } from './dto/create-account.dto';
+import { UpdateAccountDto } from './dto/update-account.dto';
+import { Account } from './entities/account.entity';
+import { ValidationPipe } from './pipes/swyft-validation.pipe';
+
```

Comment:
For swagger purposes try adding "@ApiTags('Accounts')" 
you will also need to: 
import { ApiTags } from '@nestjs/swagger';